### ISSUE TITLE
Write errors to `stderr` instead of `stdout`

### ISSUE THREAD

**Issue by aguschin** [iterative/gto#265]
After [working on gto-action](https://github.com/iterative/gto-action/pull/7) I realized that's it's inconvenient to handle this in CI. E.g. if you just want to find out the path, you do
```shell
$ export ARTIFACT_PATH=`gto describe mymodel --path`
$ echo $ARTIFACT_PATH
No path exists for artifact
```
which is in fact the error message and should be written to stderr.

The other question is that maybe `gto describe` shouldn't return exitcode 1 here at all. If the description doesn't exist, it's not an incorrect behaviour, incorrect input data or something incorrect at all. Maybe someone wants to check if annotation in `artifacts.yaml` exists at all? Then we may need another command to check it. 

Need to check this for MLEM also.

**Comment by aguschin** [comment]
UPD: confirmed and created an issue for MLEM https://github.com/iterative/mlem/issues/404

**Comment by aguschin** [comment]
Hi @itstargetconfirmed! Wanted to bring to your attention that the GTO project have the same problem as you just solved for MLEM. Do you want to create a PR for that one as well?

**Comment by itstargetconfirmed** [comment]
Hi @aguschin, I'd love to take this on. I'll create a PR tomorrow for this.

**Comment by aguschin** [comment]
closed by #276
